packages/components/src/DateTimePickers/LegacyDateTimePickers/views/DateTimeView/__snapshots__/DateTimeView.test.js.snap

```diff
@@ -50,7 +50,7 @@ exports[`DateTimeView should render 1`] = `
         onClick={[Function]}
         tabIndex="-1"
       />,
-      "middleElement": <withI18nextTranslation(HeaderTitle)
+      "middleElement": <HeaderTitle
```

Comment:
`HeaderTitle` is wrapped .... twice... in i18n HOC (in component and index file). And guess what... it doesn't use `t()`:D
I removed both wrappers